Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRabbitRpcServerTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
+    {
+        public int Value { get; set; }
+    }
+
+    class RespondStub
+    {
+        public int Value { get; set; }
+    }
+
+    public class NetqRabbitRpcServerTests
+    {
+        private NetqRabbitRpcServer _server;
+        private Mock<INetqRpcRabbitConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
+        private Mock<IBus> _busMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            _busMock = new Mock<IBus>();
+            _netqRpcRabbitConnectionMock = new Mock<INetqRpcRabbitConnection<Func<Type, object>>>();
+
+            _netqRpcRabbitConnectionMock.Setup(connection => connection.Bus).Returns(_busMock.Object);
+            Mock<ILogger> loggerMock = new Mock<ILogger>();
+            Mock<ILoggerFactory> lfMock = new Mock<ILoggerFactory>();
+            lfMock.Setup(lf => lf.CreateLogger(It.IsAny<string>())).Returns(loggerMock.Object);
+
+            _server = new NetqRabbitRpcServer(_netqRpcRabbitConnectionMock.Object, lfMock.Object);
+        }
+
+        [Test]
+        public void Respond_OnValidCallbackAndConcreteRpcType_ShouldStartRespond()
+        {
+            //Arrange
+            Func<RequestStub, RespondStub> callback = request => new RespondStub();
```

Comment:
You can make this a local function. I think it looks better, what do you think? 
```suggestion
            RespondStub Callback(RequestStub request) => new RespondStub();
```

And the same for all the stub usage

Reply:
did something better.
added two function to the class and im suing them